dao/src/main/java/com/carhouse/dao/impl/CarDaoImpl.java

```diff
@@ -146,29 +146,30 @@ public Integer addCar(final Car car) {
      * To correctly change references the method first remove old references and then add new
      *
      * @param car the car model
+     * @return check or car is update, 1 means that only one row has been updated otherwise it returns 0
```

Comment:
method returns Boolean. What are you talking about 1\0?

Reply:
I will change the return type on void because I didn't know how to throw an exception from void method and added this type just to tests, the method never returns false because an exception throws first
I will change the similar comment for delete method

Reply:
It's an interface and it's correct in general to have return type `boolean`

Reply:
Ok, I again set boolean return type for update method